src/renderer/components/basic.lua

```diff
@@ -126,6 +144,14 @@ function basicTextComponent:render(surf, position, styles, resolver)
           position.left + leftPad, cY, position.width - leftPad - rightPad,
           resolver({}, "color", styles.color), styles["text-align"] or "left", lineHeight)
       end
+    else
+      -- TODO Wrapping support?
+      local text = self.node.children[1].content or ""
+      writeBig(surf, text,
+        position.left + leftPad, cY,
+        resolver({}, "color", styles.color), bgc,
+        styles["text-align"] or "left", position.width - leftPad - rightPad)
+    elseif styles["font-size"] == "1rem" or totalSize <= 1 then
```

Comment:
This elseif case is empty.. I assume this was left in by mistake

Reply:
Honestly, I'm just as tired as you, it's 4AM here, I'll wake up able to think and fix this.